File: Lua/Common/HttpResult.lua

local this = LuaObject:New()

HttpResult = this

this.ResultType =
 {
 	['Failed'] = 'Failed',
 	['Sucess'] = 'Sucess',
}
this.ResultUnknowError = ZPLocalization.Instance.Get('HttpConnectFailed')

this.resultType = nil  
this.resultObject = nil 
this.isSwitchHost = false 

function this.HttpResult( )
	this.resultType = this.ResultType.Failed 
	this.resultObject = this.ResultUnknowError
end

File: ___HappyCityScripts/EginPlugins/Connect/HttpResult.cs
using UnityEngine;
using System.Collections;

public class HttpResult {
	
	public enum ResultType {
		Failed, 
		Sucess
	}
	
	private static string ResultUnknowError = ZPLocalization.Instance.Get("HttpConnectFailed");
	
	public ResultType resultType;
	public object resultObject;
    public bool isSwitchHost = false;
	
	public HttpResult () {
		resultType = ResultType.Failed;
		resultObject = ResultUnknowError;
	}
}
